src/bindings/c/include/c_api/ie_c_api.h

```diff
@@ -26,6 +26,7 @@
 #include <stdio.h>
 
 #include "openvino/c/deprecated.h"
+#include "ie_api.h"
```

Comment:
You cannot include headers from C++ API to C API
Please, create new macroses

Reply:
Can we add deprecation into INFERENCE_ENGINE_C_API macro itself? So each symbol is automatically deprecated